src/ext/java/org/opentripplanner/ext/transmodelapi/model/TripTimeShortHelper.java

```diff
@@ -23,6 +23,7 @@ private TripTimeShortHelper() { }
     @Nullable
     public static TripTimeShort getTripTimeShortForFromPlace(Leg leg, RoutingService routingService) {
         if (!leg.isTransitLeg()) { return null; }
+        if (leg.isFlexibleTrip()) { return null; }
```

Comment:
Below there is some code that is commented out, should that be deleted or is this just a temporary guard, in witch case there should probably be a comment here telling why we can retrieve this info on a FLEX trip.

Reply:
This is used to retrieve information to populate the estimateCall for to/from place. We have to decide if the EstimatedCallType is applicable to flexible trips. I can add a comment about it.